mirrord/agent/src/steal/http/filter.rs

```diff
@@ -88,15 +133,14 @@ pub(super) async fn filter_task(
                         .preserve_header_case(true)
                         .serve_connection(
                             reversible_stream,
-                            HyperHandler {
+                            HyperHandler::<HttpV1>::new(
```

Comment:
Can't this match statement be moved inside the enum? i.e. HttpVersion.new(...).get_connection()?

Reply:
Continuing this conversation on the new version of the code